plugins/modules/ipasudocmd.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Authors:
+#   Thomas Woerner <twoerner@redhat.com>
```

Comment:
Here the author is me, further down in DOCUMENTATION it is you.

Reply:
Fixed.